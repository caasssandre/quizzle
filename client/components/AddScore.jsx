import React from 'react'
import { connect } from 'react-redux'
import socket from '../api/socket'

class AddScore extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            team: '',
            buttonClicked: false,
            message:''
        };
    }

    handleChange = (event) => {
        this.setState({
            [event.target.name]: event.target.value
        })
    }

    submitScore = () => {
        if(this.state.buttonClicked == true){
            // do nothing
        }
        else if(this.state.team == ''){
            this.setState({
                message:'You need to enter a name for your team!'
            })
        }
        else{
            let teamScore = this.props.score.points
            socket.emit('add to leaderboard', { teamName: this.state.team, teamCode: this.props.teamName, teamSize: this.props.players.length, teamScore: teamScore, totalRounds: this.props.totalRounds })
            this.setState({
                buttonClicked:true
            })
        }
    }

    playAgain = () => {
        socket.emit('reset game', this.props.teamName)
        socket.emit('all players in', { teamName: this.props.teamName, numOfPlayers: this.props.players.length, players: this.props.players })
    }

    mainMenu = () => {
        socket.emit('reset game', this.props.teamName)
        socket.emit('main menu', this.props.teamName)
    }

    render() {
        return (
            <div className="leaderboard">
                {/* <h1 className="leaderboard-gameTitle">Quizzical</h1> */}
                <h1 className="leaderboard-title">Add Score to Leaderboard</h1>

                <p className="leaderboard-team">Team name:</p>
                <input className="leaderboard-team__field" name="team" onChange={this.handleChange} />
                <h3 className="leaderboard-team__score">Your Score: {this.props.score.points}</h3>
                <h2>{this.state.message}</h2>
                <div className='home-btns'>
                    <div className='home-btns__btn' onClick={this.submitScore}>Submit Score</div>
                </div>

                <section className='leaderboard-btnSection'>
                    <div className='setup-btns__btn' onClick={this.playAgain}>Play Again</div>
                    <div className='home-btns__btn' onClick={this.mainMenu}>Main Menu</div>
                </section>
            </div>
        )
    }
}

function mapStateToProps(state) {
    return {
        players: state.players,
        teamName: state.teamName,
        score: state.score,
        totalRounds: state.totalRounds
    }
}

export default connect(mapStateToProps)(AddScore)