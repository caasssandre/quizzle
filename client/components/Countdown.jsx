import React from 'react'
import {connect} from 'react-redux'

class Countdown extends React.Component {
  constructor(props){
    super(props)
  }

  render(){
    return (
      <div className='questions-countdown__circle'>
        <h1 className='questions-countdown'>{this.props.clock - this.props.players.length * 20}</h1>
      </div>
    )
  }
}

function mapStateToProps(state){
  return{
    clock:state.clock,
    players: state.players
  }
}

export default connect(mapStateToProps)(Countdown)