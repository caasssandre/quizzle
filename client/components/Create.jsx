import React from 'react'
import { connect } from 'react-redux'

import { addPlayerToTeam, getTeams } from '../api/users'
import socket from '../api/socket'

class Create extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      captainName:''
    }
  }
  
  componentDidMount() {
    this.generateCode()
  }

  handleChange = (event) => {
    this.setState({
      [event.target.name]: event.target.value.toUpperCase()
    })
  }
  
  getRandomUppercaseChar = () => {
    let r = Math.floor(Math.random() * 26);
    return String.fromCharCode(65 + r);
  }
  
  generateCode = () => {
    let prefix = new Array(2).fill().map(() => this.getRandomUppercaseChar()).join(""),
      integer = Math.floor((Math.random() * 9999));
    let code = prefix + integer
    getTeams().then(teams => {
      if (teams.text.includes(code)) {
        this.generateCode()
      }
      else {
        this.setState({
          team: code
        })
        return code
      }
    })
  }

  createTeam = () => {
    if(this.state.captainName == ''){
      this.setState({
        message:'Please enter a username'
      })
    }
    else{
      this.addPlayerToTeam(true)
    }
  }

  addPlayerToTeam = (captain) => {
    socket.emit('join team', this.state.team)
    addPlayerToTeam(this.state.captainName, this.state.team, captain)
      .then(players => {
        socket.emit('show players in lobby', players)
        this.props.dispatch({
          type: 'SAVE_PLAYER_DETAILS',
          playerInfo: {
            name: this.state.captainName,
            captain: captain,
            index: players.length - 1
          }
        })
      })
    this.props.dispatch({
      type: 'SAVE_TEAM_NAME',
      teamName: this.state.team
    })
    this.props.dispatch({
      type: 'INCREMENT_PAGE',
    })
  }


  render() {
    return (

      <main>
        <section className='setup'>
          <h1 className='setup-gameTitle'>Quizzical</h1>
          <h2 className='setup-create'>Game Created</h2>
          <h1>Team Code:</h1>
          <h1>{this.state.team}</h1>
          <p>Give this code to your team</p>
        </section>

        <section className='setup'>
          <p className='setup-form'>Enter your player name below:</p>
          <input name="captainName" onChange={this.handleChange} value={this.state.captainName} />

          <form>
            <div className='setup-btns'>
              <section>
                <div className='setup-btns__btn' onClick={this.createTeam}>
                  Create Team
                </div>
              </section>
            </div>
            <section>
              <p className='setup-join'>Or click below to join another team:</p>
              <div className='setup-btns__btn' onClick={(e) => this.props.changePage(e, 'join')}>
                Join Team
                </div>
            </section>
            {this.state.message != '' && <h2>{this.state.message}</h2>}
          </form>
        </section>
      </main>

    )
  }
}

export default connect()(Create)