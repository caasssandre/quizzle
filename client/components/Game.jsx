import React from 'react'
import { connect } from 'react-redux'
import { log } from 'util'
// import { getQuestions } from '../../server/routes/questions'
class Game extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      display1: '',
      display2: '',
      display3: '',
      display4: ''
    }
  }

  componentDidMount() {
    const answerArr = [
      'correctAnswer',
      'incorrectAnswer1',
      'incorrectAnswer2',
      'incorrectAnswer3'
    ]

    function randomAnswer(answerArr) {
      let length = answerArr.length
      let lastItem
      let i
      while (length) {
        i = Math.floor(Math.random() * length--)

        lastItem = answerArr[length]
        answerArr[length] = answerArr[i]
        answerArr[i] = lastItem
      }

      let randomArr = answerArr
      return randomArr
    }

    let randomArr = randomAnswer(answerArr)
    this.setState({
      display1: randomArr[0],
      display2: randomArr[1],
      display3: randomArr[2],
      display4: randomArr[3]
    })
  }

  selectAnswer = event => {
    event.preventDefault()
    this.props.dispatch({
      type: 'SUBMIT_ANSWER',
      response: {
        question: this.props.questions.jumbledTrivias[this.props.player.index]
          .question,
        correctAnswer: this.props.questions.jumbledTrivias[
          this.props.player.index
        ].correctAnswer,
        selectedAnswer: event.target.id
      }
    })
    this.setState({
      submittedAnswer: true
    })
    // this.props.dispatch({
    //   type: 'INCREMENT_PAGE'
    // })
  }

  render() {
    let q = this.props.questions
    return (
      <div>
        <h1>Game Component</h1>
        {q.trivias && <h2>{q.trivias[this.props.player.index].question}</h2>}

        {!this.state.submittedAnswer && q.jumbledTrivias && (
          <div className='btn-group'>
            <button
              id={
                q.jumbledTrivias[this.props.player.index][this.state.display1]
              }
              onClick={this.selectAnswer}
            >
              {q.jumbledTrivias[this.props.player.index][this.state.display1]}
            </button>
            <button
              id={
                q.jumbledTrivias[this.props.player.index][this.state.display2]
              }
              onClick={this.selectAnswer}
            >
              {q.jumbledTrivias[this.props.player.index][this.state.display2]}
            </button>
            <button
              id={
                q.jumbledTrivias[this.props.player.index][this.state.display3]
              }
              onClick={this.selectAnswer}
            >
              {q.jumbledTrivias[this.props.player.index][this.state.display3]}
            </button>
            <button
              id={
                q.jumbledTrivias[this.props.player.index][this.state.display4]
              }
              onClick={this.selectAnswer}
            >
              {q.jumbledTrivias[this.props.player.index][this.state.display4]}
            </button>
          </div>
        )}

        {this.state.submittedAnswer  && (
          <div className='btn-group'>
            <button>
              {this.props.playerResponses[0].selectedAnswer}
            </button>            
          </div>
        )}
      </div>
    )
  }
}

function mapStateToProps(state) {
  return {
    player: state.player,
    questions: state.questions,
    playerResponses: state.playerResponses
  }
}

export default connect(mapStateToProps)(Game)
