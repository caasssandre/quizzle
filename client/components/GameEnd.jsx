import React from 'react'
import ChartistGraph from 'react-chartist'
import { connect } from 'react-redux'
import socket from '../api/socket'

class GameEnd extends React.Component {
  constructor(props) {
    super(props)
  }

  playAgain = () => {
    socket.emit('reset game', this.props.teamName)
    socket.emit('all players in', { teamName: this.props.teamName, numOfPlayers: this.props.players.length })
  }

  mainMenu = () => {
    socket.emit('reset game', this.props.teamName)
    socket.emit('main menu', this.props.teamName)
  }

  leaderboard = () => {
    socket.emit('increment pages', this.props.teamName)
  }

  render() {
    let options = {
      labelInterpolationFnc: function (value, index) {
        return Math.round(value / data.series.reduce(sum) * 100) + '% ' + data.label[index];
      }
    }
    let sum = function (a, b) { return a + b }
    let data = { label: ['Right', 'Wrong'], series: [this.props.score.correct, this.props.score.total - this.props.score.correct] }

    return (
      <div className='end'>
        <h1 className='end-gameTitle'>Quizzical</h1>
        <h1 className='end-title'>
          Congrats!
        </h1>
        <ChartistGraph className='ct-chart' data={data} options={options} type={'Pie'} />
        <h3 className='end-scoreText'>
          Your team got {this.props.score.correct} out of{' '}
          {this.props.score.total} answers correct!
        </h3>

        {this.props.player.captain && (
          <div className='home-btns'>
            <div className='home-btns__btn' onClick={this.playAgain}>Play Again!!</div>
            <div className='home-btns__btn' onClick={this.mainMenu}>Main Menu</div>
            <div className='home-btns__btn' onClick={this.leaderboard}>Leaderboard</div>
          </div>
        )



        /* <div className='end-btns'>
          {this.props.player.captain && (
            <div className='end-btns__btn' onClick={this.playAgain}>
              Play again!!
            </div>
          )}
          {this.props.player.captain && (
            <div className='end-btns__btn' onClick={this.mainMenu}>
              Main Menu
            </div>
          )}
        </div> */}
        {/* {this.props.player.captain && ( */}
        {/* <div className='end-btns'> */}
        {/* <h3>Keen to see how you measured up?</h3> */}
        {/* <div className='end-btns__btn' onClick={this.leaderboard}> */}
        {/* Leaderboard */}
        {/* </div> */}
        {/* </div> */}
        {/* )}  */}
      </div>
    )
  }
}

function mapStateToProps(state) {
  return {
    teamName: state.teamName,
    player: state.player,
    players: state.players,
    score: state.score
  }
}

export default connect(mapStateToProps)(GameEnd)