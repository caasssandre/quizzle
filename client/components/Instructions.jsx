import React from 'react'
import {connect} from 'react-redux'

export class Instructions extends React.Component {
    constructor(props) {
        super(props)
        this.state = {
        }
    }


    render() {
        return (
          <main>
            <section className='home'>
              <h1 className='home-gameTitle'>Quizzical</h1>
              <div className='instruct'>
                <h3 className='instruct-subTitle'>Instructions:</h3>
                <div className='instruct-info'>
                  <p className='instruct-info__step'>Step 1:</p>
                  <p>
                    Create your team! One player needs to go to the 'Start Game'
                    page, and create a new game by entering in the team name and
                    their username.
                  </p>
                  <p className='instruct-info__step'>Step 2:</p>
                  <p>
                    Time for everyone else to join! All other players go to the
                    'Start Game' page and enter in the team name and their
                    username.
                  </p>
                  <p className='instruct-info__step'>Step 3: </p>
                  <p>
                    Check everyone is in! All players currently in game will
                    display on the lobby page. Once everyone has joined, the
                    team captain (the player who created the team) clicks the
                    'Start Game' button.
                  </p>
                  <p className='instruct-info__step'>Step 4: </p>
                  <p>
                    Time to get answering questions. Each user will be able to
                    see one question, and a list of four potential answers. But
                    here's the catch... your question does not match the answers
                    you can see (or maybe it does... who knows?!)! Your task -
                    find the player whose answers match your question, and
                    figure out the correct answer! You'll need to be quick about
                    it thought - and answer before the time runs out!
                  </p>
                </div>
                <div className='home-btns__btn' onClick={(e)=>this.props.changePage(e, 'main')}>
                  Go Back
                </div>
              </div>
            </section>
          </main>
        )
    }
}

function mapStateToProps(state) {
    return {
        room: state.roomName,
    }
}

export default connect(mapStateToProps)(Instructions)