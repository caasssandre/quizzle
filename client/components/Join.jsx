import React from 'react'
import { connect } from 'react-redux'

import {savePlayerDetails, incrementPage, saveTeamName} from '../actions'
import { addPlayerToTeam, getTeams, getPlayersByTeam } from '../api/users'
import socket from '../api/socket'

class Join extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      player:'',
      team:''
    }
  }

  handleChange = (event) => {
    this.setState({
      [event.target.name]: event.target.value.toUpperCase()
    })
  }

  getRandomUppercaseChar = () => {
    let r = Math.floor(Math.random() * 26);
    return String.fromCharCode(65 + r);
  }


  joinTeam = () => {
    getTeams().then(res => {
      
      this.setState({
        message: ''
      })
      if(this.state.team == ''){
        this.setState({
          message:'Please enter a valid team code'
        })
      }
      else if(this.state.player == ''){
        this.setState({
          message:'Please enter a username'
        })
      }
      else if (!res.text.includes(this.state.team)) {
        this.setState({
          message: 'This team does not exist, maybe you would like to create one?'
        })
      }
      else {
        getPlayersByTeam(this.state.team).then(res => {
          if(JSON.parse(res.text)[0].game_started){
            this.setState({
              message: 'This team has started playing without you!'
            })
          }
          else if (!JSON.parse(res.text).find(player => {
            return player.name == this.state.player
          })) {
            this.addPlayerToTeam(false)
          }
          else {
            this.setState({
              message: 'This username is taken - please pick a new one.'
            })
          }
        })
      }
    })
  }

  addPlayerToTeam = (captain) => {
    socket.emit('join team', this.state.team)
    addPlayerToTeam(this.state.player, this.state.team, captain, this.props.player.socketId)
      .then(players => {
        socket.emit('show players in lobby', players)
        this.props.dispatch(savePlayerDetails(this.state.player, captain, players.length-1)
        )
      })
    this.props.dispatch(saveTeamName(this.state.team))
    this.props.dispatch(incrementPage())
  }

  render() {
    return (
      <main>
        <section className='setup'>
          <h1 className='setup-gameTitle'>Quizzical</h1>
          <form>
            <section className='setup-team'>
              <p className="setup-team__text">Team Code:</p>
              <input
                className='setup-team__fields'
                type='text'
                name='team'
                onChange={this.handleChange}
                value={this.state.team}
              />
            </section>
            <section className='setup-user'>
              <p className="setup-user__text">User Name:</p>
              <input
                className='setup-user__fields'
                type='text'
                name='player'
                onChange={this.handleChange}
                value={this.state.player}
              />
            </section>
            <section>
              <div className='setup-btns__btn' onClick={this.joinTeam}>
                Join Team
                </div>
            </section>
            <section className='setup-join'>
              <p>Not quite what you want?</p>
              <div className='setup-btns__btn' onClick={(e) => this.props.changePage(e, 'create')}>
                Create Team
                </div>
                <div className='setup-btns__btn' onClick={(e) => this.props.changePage(e, 'instructions')}>
                Rules
                </div>
                <div className='setup-btns__btn' onClick={(e)=>this.props.changePage(e, 'main')}>
                  Main menu
                </div>
            </section>
            {this.state.message != '' && <h2>{this.state.message}</h2>}
          </form>
        </section>
      </main>
    )
  }
}

function mapStateToProps(state){
  return{
    player: state.player
  }
}

export default connect(mapStateToProps)(Join)