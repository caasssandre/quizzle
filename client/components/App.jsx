import React from 'react'
import { connect } from 'react-redux'
import { HashRouter as Router } from 'react-router-dom'

import socket from '../api/socket'

import Welcome from './Welcome'
import Game from './Game'
import Results from './Results'
import GameEnd from './GameEnd'
import Lobby from './Lobby'
import Leaderboard from './Leaderboard'
import StopGame from './StopGame'

import { saveSocketId } from '../actions/index'
import { goToGame, goToMainMenu, incrementPage, goToStopGame } from '../actions/index'
import { addQuestions, resetQuestions } from '../actions/index'
import { resetPlayerResponses } from '../actions/index'
import { incrementAnswerCount, resetAnswerCount } from '../actions/index'
import { resetClock, decrementClock } from '../actions/index'
import { incrementScore, resetScore, saveStrike, resetStrike, saveStreak } from '../actions/index'
import { incrementRound, resetRound, setTotalRounds } from '../actions/index'
import { addLeaderboard, resetLeaderboard } from '../actions/index'
import { addPlayers, removeMissingPlayers } from '../actions/index'
import { addToMissingPlayers, resetMissingPlayers } from '../actions/index'

const cooldownfx = "/sfx/cooldown2.mp3"
const cooldown = new Audio(cooldownfx);

export class App extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      roundScores: []
    }
  }

  componentDidMount() {
    // Handle browser navigation    
    window.addEventListener('popstate', () => {
      history.pushState(null, null, location.href)
      history.go(1)
    })

    // Handle phone not going to sleep
    const noSleep = new NoSleep()
    document.addEventListener('touchstart', function () {
      noSleep.enable()
    })

    // Receives socket id from server, adds to state
    socket.on('send id', id => {
      this.props.dispatch(saveSocketId(id))
    })

    // Stops game when another player leaves the team
    socket.on('user has left team', player => {
      this.props.dispatch(addToMissingPlayers(player.name))
      if (this.props.pageNumber <= 4) {
        if (!this.props.missingPlayers.includes(this.props.player.name)) {
          this.props.dispatch(goToStopGame())
        }
      }
    })

    // Reset game
    socket.on('reset game', () => {
      this.props.dispatch(resetQuestions())
      this.props.dispatch(resetPlayerResponses())
      this.props.dispatch(resetAnswerCount())
      this.props.dispatch(resetRound())
      this.props.dispatch(resetScore())
      this.props.dispatch(resetLeaderboard())
      this.props.dispatch(resetStrike())
    })

    // Increment page
    socket.on('increment pages', () => {
      this.props.dispatch(incrementPage())
    })

    // Reset Game - back to main menu
    socket.on('main menu', () => {
      this.props.dispatch(goToMainMenu())
    })

    // When back-end receives 'all players in', it makes the api call to get new questions
    socket.on('all players in', (players) => {
      if (this.props.missingPlayers.length != 0) {
        this.props.dispatch(removeMissingPlayers(this.props.missingPlayers))
        this.props.dispatch(addPlayers(this.props.players))
        this.props.dispatch(resetMissingPlayers())
      }
      else {
        this.props.dispatch(addPlayers(players))
      }
      this.props.dispatch(goToGame())
    })

    // Set total rounds
    socket.on('receive total rounds', totalRounds => {
      this.props.dispatch(setTotalRounds(totalRounds))
    })

    // Prepare game to start new round
    // When back-end receives 'new question', it makes the api call to get new questions
    socket.on('new question', () => {
      this.props.dispatch(resetAnswerCount())
      this.props.dispatch(resetQuestions())
      this.props.dispatch(resetPlayerResponses())
      this.props.dispatch(incrementRound())
      this.props.dispatch(goToGame())
    })

    // Receives and sets questions array from API call, and starts the timer
    socket.on('receive questions', questions => {
      this.props.dispatch(resetClock(this.props.players.length))
      this.props.dispatch(addQuestions(questions))
      cooldown.play()

      this.interval = setInterval(() => {
        if (this.props.clock == 0 || this.props.pageNumber != 3) {
          clearInterval(this.interval)
          this.props.dispatch(resetClock(this.props.players.length))
        }
        else { this.props.dispatch(decrementClock()) }
      }, 1000)
    })

    // Handle answer count
    socket.on('submitted answer', () => {
      this.props.dispatch(incrementAnswerCount())
    })

    // Handle score
    socket.on('score', score => {
      this.setState({
        roundScores: [...this.state.roundScores, score]
      })
      this.props.dispatch(incrementScore(score))
    })

    socket.on('check for strike', () => {
      if (!this.state.roundScores.includes(0)) {
        this.props.dispatch(saveStrike(1))
        this.props.dispatch(saveStreak(this.props.strikeCount))
      }
      else {
        this.props.dispatch(saveStrike(0))
      }
      this.setState({
        roundScores: []
      })
    })

    // Leaderboard
    socket.on('receive leaderboard', leaderboard => {
      this.props.dispatch(addLeaderboard(leaderboard))
    })
  }

  render() {
    return (
      <Router>
        {this.props.pageNumber != 1 &&
        <div className="gameTitle-box">
          <h1 className='gameTitle-title'>Quizzical</h1>
        </div>
        }
        {this.props.pageNumber == 1 && <Welcome />}
        {this.props.pageNumber == 2 && <Lobby />}
        {this.props.pageNumber == 3 && <Game />}
        {this.props.pageNumber == 4 && <Results strike={!this.state.roundScores.includes(0)} />}
        {this.props.pageNumber == 5 && <GameEnd />}
        {this.props.pageNumber == 6 && <Leaderboard />}
        {this.props.pageNumber == 7 && <StopGame />}
      </Router>
    )
  }
}

function mapStateToProps(state) {
  return {
    pageNumber: state.pageNumber,
    clock: state.clock,
    players: state.players,
    strikeCount: state.strikeCount,
    player: state.player,
    missingPlayers: state.missingPlayers
  }
}

export default connect(mapStateToProps)(App)
