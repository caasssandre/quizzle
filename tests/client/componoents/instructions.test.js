import { Instructions } from '../../../client/components/Instructions'
import React from 'react'
import { shallow, mount } from 'enzyme'


// Tests to see that the Instructions component renders
test('test to see if instructions page renders', () => {
    const component = shallow(<Instructions/>)
  
    expect(component).toMatchSnapshot()
  
  })


  
// test('test state', () => {
//     const component = mount(<Instructions />);
//     component.find('[className="home-btns__btn"]').simulate('click');
//     expect(component.state('isLoading')).toEqual(true);
//     component.unmount();
//   });

