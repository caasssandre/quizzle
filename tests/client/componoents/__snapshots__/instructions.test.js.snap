// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to see if instructions page renders 1`] = `ShallowWrapper {}`;
